test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
@harshgMSFT - the javascript is posting the xsrf in this format right now (cookietoken:formtoken). Will posting the same format work with the attribute or is it required in a different format?